firmware/controllers/gauges/tachometer.cpp

```diff
@@ -18,26 +18,71 @@
 EXTERN_ENGINE;
 
 static scheduling_s tachTurnSignalOff;
+static float halfperiodMs = 4; /* period in ms, used for multi pulse output, actually half period */
+static uint8_t isStarted = 0;
+
+static void toggleTachPinHigh(void);
+static void toggleTachPinLow(void);
 
 static void turnTachPinLow(void) {
 	enginePins.tachOut.setLow();
 }
 
+static void toggleTachPinLow(void)
+{
+	if(isStarted)
+	{
+		enginePins.tachOut.setLow();
+		engine->executor.scheduleForLater(&tachTurnSignalOff, (int)MS2US(halfperiodMs), (schfunc_t) &toggleTachPinHigh, NULL);
+	}
+}
+
+static void toggleTachPinHigh(void)
+{
+	if(isStarted)
```

Comment:
code style - space is missing but new line is not consistent with project code style

Reply:
Sorry but i do not get it, line number pls.

Reply:
```
if (condition) {
    ...
}
```